.github/workflows/check_reference.yaml

```diff
@@ -0,0 +1,33 @@
+name: Check reference strings
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/check_reference_strings.yaml'
+      - '.github/scripts/linter_config.json'
+      - 'en-US/*.xliff'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/check_reference_strings.yaml'
```

Comment:
Same as above. Recommend updating the filename to `check_reference_strings.yaml` to match.